### PR TITLE
Fix profile image for recent sessions

### PR DESCRIPTION
**Pull request by jonasfj**
Currently, if I do a sign-out / sign-in, the profile image is broken.
Probably something changed in auth APIs that started inserting default parameters.
That makes sense, but it breaks when we then add additional parameters.

### CODE REVIEW

File: app/lib/account/models.dart [hunk 1]
```diff
@@ -199,7 +199,13 @@ class UserSessionData {
     }
     // Double the layout size, for better quality on higher dpi displays.
     final imageSize = layoutSize * 2;
-    return '$imageUrl=s$imageSize';
+
+    // Strip existing options from the imageUrl if there is any
+    var u = imageUrl;
+    if (RegExp(r'=(?:[swh]\d+)|[cp](?:-(?:[swh]\d+)|[cp])*$').hasMatch(u)) {
```
Comment (isoos) [inline app/lib/account/models.dart:205]: Where is this regexp coming from?

Comment (jonasfj) [reply app/lib/account/models.dart:205]: I wrote it to match the format in https://developers.google.com/people/image-sizing

Comment (isoos) [reply app/lib/account/models.dart:205]: Please note the url in a comment.

Comment (jonasfj) [reply app/lib/account/models.dart]: the URL is higher up, but yes, moving it out I'll duplicate.

**Review by isoos — APPROVED** [review]

**Pull request merged**